cloud/amazon/iam.py

```diff
@@ -53,6 +53,17 @@
       - When creating or updating, specify the desired path of the resource. If state is present, it will replace the current path to match what is passed in when they do not match.
     required: false
     default: "/"
+  trust_policy:
+    description:
+      - The inline (JSON or YAML) trust policy document that grants an entity permission to assume the role. Mutually exclusive with C(trust_policy_filepath).
+    required: false
+    default: null
+  trust_policy_filepath:
+    description:
+      - The path to the trust policy document that grants an entity permission to assume the role. Mutually exclusive with C(trust_policy).
+    required: false
+    default: null
+
```

Comment:
Ah yes for added parameters... ok, will do.

On 17 March 2016 at 16:18, Michael Scherer <notifications@github.com> wrote:

> In cloud/amazon/iam.py
> <https://github.com/ansible/ansible-modules-core/pull/3264#discussion_r56570560>
> :
>
> > @@ -53,6 +53,17 @@
> >        - When creating or updating, specify the desired path of the resource. If state is present, it will replace the current path to match what is passed in when they do not match.
> >      required: false
> >      default: "/"
> > +  trust_policy:
> > +    description:
> > +      - The inline (JSON or YAML) trust policy document that grants an entity permission to assume the role. Mutually exclusive with C(trust_policy_filepath).
> > +    required: false
> > +    default: null
> > +  trust_policy_filepath:
> > +    description:
> > +      - The path to the trust policy document that grants an entity permission to assume the role. Mutually exclusive with C(trust_policy).
> > +    required: false
> > +    default: null
> > +
>
> Could you add 'version_added' (see other module) ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ansible/ansible-modules-core/pull/3264/files/d1b8affd96e07afc984923c78a04c57329963b70#r56570560>
>



-- 
Pierre Jodouin
Cloud Solutions Architect